src/components/sign-up/sign-up.jsx

```diff
@@ -21,7 +21,7 @@ export default () => {
       const sendUserDetails = { ...user };
       delete sendUserDetails.confirmPassword;
       fetch(`${endpointUrl}/sign-up/${token}`, {
-        method: "POST",
+        credentials: "include",
```

Comment:
This still needs to be a POST

And we don't need to send the cookie here, so you can remove `credentials: "include"`